doc/en/skipping.rst

```diff
@@ -293,6 +293,18 @@ imperatively, in test or setup code::
             # or
             pytest.skip("unsupported configuration")
 
+Note that calling ``pytest.skip`` at the module level 
+is not allowed since pytest 3.0. To skip
+all tests in a module given some runtime condition, you can
+set a ``pytestmark`` variable:
+
+.. code-block:: python
+
+    if SOME_CONDITION:
```

Comment:
I think I'm not getting my point across.

The idea of this addition was just to offer a quick replacement for someone using `pytest.skip` at the module level. Without looking at specifics, the answer would be:

```python
# if for some reason you're doing this at the module level
pytest.skip('some reason')
# you should replace the above by this:
pytestmark = pytest.mark.skip('some reason')
```

IOW, how to update the code without changing too much. People might be interested in the quick solution in case they have hundreds of cases like this in a code base and want to do a quick find/replace.

But I agree we probably should have a section with best practices on how to skip tests at module level, for example:

```python
# 1. skip all tests in a module unconditionally
pytestmark = pytest.mark.skip('all tests still WIP')

# 2. skip all tests in a module based on some condition
pytestmark = pytest.mark.skipif(sys.platform == 'win32', 'tests for linux only')

# 3. skip all tests in a module if some import is missing
pexpect = pytest.importorskip('pexpect')
```